content/vendor/cli/tokens.md

```diff
@@ -17,11 +17,15 @@ Tokens may only access a portion of the Vendor API. The following administrative
 
 Tokens are primarily used with [Vendor CLI](https://help.replicated.com/api/replicated-vendor-cli/) or [REST API](https://help.replicated.com/api/vendor-api/) commands for automated customer, channel, and release management.
 
-## Team Tokens
+## Service Accounts
 
-Team tokens are available to all members of the vendor team. These tokens can be created, retrieved, and revoked by any user with the proper RBAC policy.
+Service accounts are assigned a token and associated with an RBAC policy. Any user with the proper permissions can create, retrieve or revoke them.
```

Comment:
Should we include a description here on how RBAC selection works?
Something like:
```
Non-Admin users can create a Service Account with their own RBAC policy applied.  Admin users can select any RBAC policy for a new Service Account.
```

Reply:
i already added something like that on the line below but i should probably move it up here to make it more noticeable

Reply:
ah I don't know why I didn't see the line below